tfhe/src/js_on_wasm_api/mod.rs

```diff
@@ -1,15 +1,15 @@
 #[cfg(feature = "shortint-client-js-wasm-api")]
-pub mod shortint;
+mod shortint;
 #[cfg(feature = "shortint-client-js-wasm-api")]
 pub use shortint::*;
 
 #[cfg(feature = "boolean-client-js-wasm-api")]
-pub mod boolean;
+mod boolean;
 #[cfg(feature = "boolean-client-js-wasm-api")]
 pub use boolean::*;
 
 #[cfg(feature = "parallel-wasm-api")]
 pub use wasm_bindgen_rayon::init_thread_pool;
 
-pub mod high_level_api;
-pub use high_level_api::*;
+mod js_high_level_api;
+pub use js_high_level_api::*;
```

Comment:
how is this legal for the mod and pub use ?

Reply:
If a private item is reexported publicly (with a pub use), it can be accessed (only) under the new name (https://doc.rust-lang.org/reference/visibility-and-privacy.html#re-exporting-and-visibility)

I removed the `pub` in front of `mod` to have a simpler public interface

Reply:
all good then !